insolar/gen/reference.go

```diff
@@ -50,3 +55,8 @@ func Reference() (ref insolar.Reference) {
 	fuzz.New().NilChance(0).Fuzz(&ref)
 	return
 }
+
+func ReferencePointer() *insolar.Reference {
```

Comment:
There is no point to making a function for that.

Reply:
OK, agree.